build/DependencyVersions.props

```diff
@@ -11,7 +11,7 @@
 
   <PropertyGroup>
     <!-- Dependencies from https://github.com/aspnet/AspNetCore -->
-    <MicrosoftAspNetCoreAppPackageVersion>3.0.0-preview-18572-0018</MicrosoftAspNetCoreAppPackageVersion>
```

Comment:
just want to make sure the version number looks lower is it expected?

Reply:
Nope, not expected - tks for catching that.